DirectEmployers/MyJobs
# [PD-3720] timeline request resets drilldown

## What's this PR do?
PR keeps the drilldown as is when the timeline is activated

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside site traffic of analytics where the multifilters are selectable

## How should this be manually tested?
Pull down the repository. Go inside of site-traffic and go to a multifilter drilldown. Activate more of the drilldowns and then activate the timeline above the charts. The multifilters should still be selected and not reset back to just one

## Any background context you want to provide?
